File: Gawake/backend.py
#!/usr/bin/python
# -*- coding: utf-8
import os

"""
Estrutura da array:
    ======================================================================
    ||   Nome da regra   |   Dias(7)    |   Horário    |    Comando     ||
    ======================================================================
"""

rules = []

rules.append(['Regra 1', t, 'mon', 'tue', 'wed', 'thu', 'fri', 'sat', '19:30', 'firefox'])
rules.append(['Regra 2', 'sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat', '19:30', 'firefox'])

l = len(rules)
print(l)

for r in rules:
  print(r)

rules.pop(1)
print('')

l = len(rules)
print(l)

for r in rules:
  print(r)

os.system('echo "teste"')

File: Gawake/backend.c
#include <stdio.h>
#include <stdlib.h>

/* 
    Estrutura da array:
    ____________________________________________________________________
    |   Nome da regra   |   Dias(7)    |   Horário    |    Comando     |

 */

int main(void){
    char rules[99][9][99] = {"1234567891", "1234567891", "1234567891", "1234567891", "1234567891", "1234567891", "1234567891", "1234567891", "1234567891"};
    int i;
    int j;

    //for(i = 0; i < 1; i++){
        for(j = 0; j < 10; j++){
            printf("O valor da coluna [%d] é %c\n", j, rules[j]);
        }
    //}
}